### ISSUE TITLE
Network tests failing.

### ISSUE THREAD

**Issue by thecodefactory** [libbitcoin/libbitcoin-explorer#602]
Just realized that network tests have been disabled for some time and should be re-enabled.

**Comment by evoskuil** [comment]
Yes/no. Network tests are not reliable as unit tests.

**Comment by thecodefactory** [comment]
Agreed, also required is a version4 community server.  But I at least want to enable it to ensure the tests can complete in testing, so more a reminder.

**Comment by evoskuil** [comment]
Changing this to a bug. Network tests are still failing.

**Comment by thecodefactory** [comment]
> Changing this to a bug. Network tests are still failing.

IIRC, many of the failing tests require payment indexing and a v4 server at this point.  I agree it's a bug, but maybe temporarily we can allocate some additional domain pointers for v4 mainnet/testnet?  Speaking of which, is there are a plan to deprecate v3 mainnet/testnet domains while transitioning over, or will they just be replaced at some specified point?

**Comment by evoskuil** [comment]
Sorry, kind of rushing through triage and forgot what the deal was here. We’ll just replace the community servers on release - after an announcement to the lists.

**Comment by thecodefactory** [comment]
Ping on network tests being broken.  We need v4 mainnet/testnet servers up for these to work (or identify if they're not working yet).  I think we have some mainnet v4 servers coming online at some point in the future, can look into.

**Comment by evoskuil** [comment]
Need to get v4 server out first, working on it.
